### PR TITLE
Bug 1764353 Remove namespace and other minor updates

### PR DESCRIPTION
**Pull request by bmcelvee**
https://bugzilla.redhat.com/show_bug.cgi?id=1764353

### CODE REVIEW

**Comment by bmcelvee** [comment]
@openshift/team-documentation PTAL, thanks!

File: modules/samples-operator-crd.adoc [hunk 1]
```diff
@@ -4,25 +4,25 @@
 
 
 [id="samples-operator-crd{context}"]
-= Accessing the Samples Operator file
+= Accessing the Samples Operator configuration
 
 You can configure the Samples Operator by editing the file with the provided
 parameters.
 
 .Procedure
```
Comment (kalexand-rh) [inline modules/samples-operator-crd.adoc]: I admit that it's out of scope, but you should have prereqs about installing oc and whatever permissions, if any, you need to run this command.

File: modules/samples-operator-crd.adoc [hunk 2]
```diff
@@ -4,25 +4,25 @@
...
-*  Access the Samples Operator file:
+*  Access the Samples Operator configuration:
 +
 ----
-# oc get configs.samples.operator.openshift.io -n openshift-cluster-samples-operator
+$ oc get configs.samples.operator.openshift.io/cluster -o yaml
 ----
 +
-The following is an example of the Samples Operator file:
+The following is an example of the Samples Operator configuration:
```
Comment (kalexand-rh) [inline modules/samples-operator-crd.adoc]: You need a noun after "following," but I'd probably say "The Samples Operator configuration resembles the following example:" instead.

> :no_entry_sign: following n
Do not use as a noun except to refer to a group of followers, adherents, or partisans. For example, do not
write “Do the following” or “The functions include the following” as lead-in wording to a list. Use
“following” only as an adjective, as in “Complete the following steps” or “The adapter has the following
functions.”

**Review by kalexand-rh — COMMENTED** [review]
I have one style pick, but it otherwise LGTM.

**Comment by bmcelvee** [comment]
No QE review required.

**Pull request merged**